src/signals_notebook/users/user.py

```diff
@@ -0,0 +1,183 @@
+        """Create new User
+
+        Returns:
+            User
+        """
+        api = SignalsNotebookApi.get_default_api()
+        log.debug('Create User: %s...', cls.__name__)
+        response = api.call(
+            method='POST',
+            path=(cls._get_endpoint(),),
+            json={
+                'data': {
+                    'attributes': request.dict(by_alias=True, exclude_none=True),
+                }
+            },
+        )
+        log.debug('User: %s was created.', cls.__name__)
+
+        result = UserResponse(**response.json())
+        return cast(ResponseData, result.data).body
+
+    def refresh(self) -> None:
+        """Refresh user with new changes values
+
+        Returns:
+
+        """
+        from signals_notebook.users import UserStore
+
+        UserStore.refresh(self)
+
+    def save(self) -> None:
+        """Update attributes of a specified user.
+
+        Returns:
+
+        """
+        api = SignalsNotebookApi.get_default_api()
+        api.call(
+            method='PATCH',
+            path=(
+                self._get_endpoint(),
+                self.id,
+            ),
+            json={
+                'data': {
+                    'attributes': self.dict(
+                        by_alias=True,
+                        include={
+                            'alias',
+                            'country',
+                            'first_name',
+                            'last_name',
+                            'organization',
+                        },
+                    )
+                },
+            },
+        )
+
+    def delete(self):
+        """Make specified user disabled.
+
+        Returns:
+
+        """
+        api = SignalsNotebookApi.get_default_api()
+        log.debug('Disable User: %s...', self.id)
+
+        api.call(
+            method='DELETE',
+            path=(self._get_endpoint(), self.id),
+        )
+        log.debug('User: %s was disabled successfully', self.id)
+
+    @property
+    def picture(self) -> Optional[File]:
+        if self._picture:
+            return self._picture
+
+        api = SignalsNotebookApi.get_default_api()
+        response = api.call(
+            method='GET',
+            path=(self._get_endpoint(), self.id, 'picture'),
+        )
+        if response.content == b'':
+            return self._picture
+
+        content_disposition = response.headers.get('content-disposition', '')
+        content_type = response.headers.get('content-type', '')
+        extension = mimetypes.guess_extension(content_type)
+
+        _, params = cgi.parse_header(content_disposition)
+        file_name = f'{self.first_name}_{self.last_name}{extension}'
+        self._picture = File(name=file_name, content=response.content, content_type=content_type)
+        return self._picture
+
+    @property
+    def groups(self):
+        return self.get_system_groups()
+
+    def get_system_groups(self):
```

Comment:
useless 
U have double functional for this class

- _groups attribute
- groups property
- and this method

Create lazy load and for this class (maybe only for groups/ maybe for relationships. I dont know what of this is more important for us and for lazy load logic)